contracts/staking/RewardDelegators.sol

```diff
@@ -246,6 +252,16 @@ contract RewardDelegators is
 
             _aggregateReward = _aggregateReward + _reward;
         }
+        uint256 totalDelegations = _getTotalDelegations(_cluster);
+
+        // if total delegation is more than 0.5 million pond, than insert into selector
+        if(totalDelegations >= thresholdForSelection){
```

Comment:
The threshold is that cluster should have at least 0.5 MPond. POND balance doesn't matter